migrations/v0.14/go.mod

```diff
@@ -0,0 +1,47 @@
+module github.com/go-vela/community/migrations/v0.14
+
+go 1.17
+
+require (
+	github.com/go-vela/server v0.12.1
+	github.com/go-vela/types v0.12.0
```

Comment:
Why v0.12?

Reply:
Copy pasta :shame: - when I tested it didn't affect anything but I'd rather update that. Thanks for catching it.